rpistart.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash 
+
+
+set -e 
+
+
+cd "$(dirname "$0")"
+
+
+ARCH=arm64
+ROOT_DIR=/home/ubuntu
+NUM_CORES=${NUM_CORES:-$(nproc)}
+IMAGE_FILE=""
+KERNEL_VERSION="linux-6.1.34"
+
+
+# First step: Install the required packages. These include cross-compilers for arm64
+# and required packages for QEMU itself. 
+# NOTE: We probably don't need to install `qemu-system-gui`
+sudo apt install \
```

Comment:
```suggestion
sudo apt install -y \
```
If this is meant to run non-interactively it should have `-y`.  And probably set the environment variable `DEBIAN_FRONTEND=noninteractive` so it doesn't ask questions either.